proof-templates.json

```diff
@@ -346,6 +357,13 @@
                         "Member Status"
                      ],
                      "restrictions": [
+                        {
+                           "schema_id": "QzLYGuAebsy3MXQ6b1sFiT:2:legal-professional:1.0",
+                           "issuer_did": "QzLYGuAebsy3MXQ6b1sFiT"
+                        },
+                        {
+                           "cred_def_id": "QzLYGuAebsy3MXQ6b1sFiT:3:CL:2351:lawyer"
```

Comment:
I am not sure having `{schema_id, issuer_did}` and `{cred_def_id}` makes sense. Each element of the restrictions array in an OR, and each attribute of an item is an AND. See [AnonCreds spec](https://hyperledger.github.io/anoncreds-spec/#restrictions) for restrictions